docs/user-guide/zos-components-installation-checklist.md

```diff
@@ -1,33 +1,64 @@
 # Zowe z/OS components installation checklist
 
-Follow the steps in this checklist to guide you through the installation of Zowe z/OS Components for production purposes. Use the steps in this checklist if you want to run Zowe securely and with high-availability.
+Use this checklist to guide you through the installation and configuration of Zowe server-side components for z/OS. 
 
 
-## Planning and preparing for the installation
+## Preparing for installation
 | Task | Results | Time Estimate |  
 |----|-----------------------|---|
-| [Review the Zowe installation roadmap](../user-guide/install-zos.md) | Understand the basic installation stages |    25 minutes   |                        
-| [Plan the Installation](../user-guide/installandconfig.md)| Plan the installation and address USS considerations for Zowe |  25 minutes                    | 
-| [Address System requirements](../user-guide/installandconfig.md)| System requirements are addressed: <br /> &nbsp;&nbsp;&nbsp;&nbsp;* [z/OS](../user-guide/systemrequirements-zos.md#zos)   <br />&nbsp;&nbsp;&nbsp;&nbsp;* [Node.js](../user-guide/systemrequirements-zos.md#nodejs)  <br /> &nbsp;&nbsp;&nbsp;&nbsp;* [Java](../user-guide/systemrequirements-zos.md#java)  <br /> &nbsp;&nbsp;&nbsp;&nbsp;* [z/OSMF](../user-guide/systemrequirements-zos.md#zosmf-optional) <br />&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(recommended for full functionality)<br />&nbsp;&nbsp;&nbsp;&nbsp;* [z/OSMF HA](../user-guide/zowe-ha-overview.md)<br />&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(required for production)            |  1 day                     | 
+| [Review the Zowe server-side installation overview](../user-guide/install-zos.md) | Understand the basic installation stages and the roles and responsibilities to perform the installation |    25 minutes   |                        
+| [Prepare for installation](./installandconfig)| Understand the key-concepts in server-side installation |  25 minutes                    | 
```

Comment:
`[Review the Zowe server-side installation overview](../user-guide/install-zos.md)`
`[Prepare for installation](./installandconfig)`
These two links are leading to the article parent topics, so the actions sequence looks odd